src/test/tests/unit/convert2to3.py

```diff
@@ -11,9 +11,23 @@
 
 
 
-test_script = "print 'Old Style Print!'\n"
+test_script_src = "print 'Old Style Print!'\n"
 
-TestText("simple_2to3_input",test_script)
-TestText("simple_2to3_result",ConvertPy2to3(test_script))
+TestText("simple_2to3_input",test_script_src)
+TestText("simple_2to3_result",ConvertPy2to3(test_script_src))
+
+
+#
+# Test on-the-fly logic for visit.Source
+#
+
+test_script_file = "tgen_test_py2_script.py";
+open(test_script_file,"w").write(test_script_src)
+
+curr_val = visit_utils.builtin.GetAutoPy2to3()
+
+visit_utils.SetAutoPy2to3(True)
+visit.Source(test_script_file)
+visit_utils.SetAutoPy2to3(curr_val)
 
 Exit()
```

Comment:
It says there is no newline at the end of the file. Don't know if this needs fixing, but I'm pointing it out.

Reply:
added newline